src/NosCore.GameObject/Character.cs

```diff
@@ -729,8 +732,8 @@ private NInvPacket BuyFrom(ShopItem item, short amount, short slotChar)
                 Shop.ShopItems.TryRemove(slot, out _);
             }
 
-            SendPacket(itemInstance.GeneratePocketChange((PocketType)type, slotChar));
-            SendPacket(new SMemoPacket
+            await SendPacket(itemInstance.GeneratePocketChange((PocketType)type, slotChar));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)